frontend/containers/account-type-selector/account-type-item/component.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC, useState } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+
+import { usePress, useFocusWithin } from '@react-aria/interactions';
+import noop from 'lodash-es';
+
+import { AccountTypeItemProps } from './types';
+
+export const AccountTypeItem: FC<AccountTypeItemProps> = ({
+  accountTypeId,
+  name,
+  description,
+  imageSrc,
+  imageTitle,
+  onClick = noop,
+}: AccountTypeItemProps) => {
+  const [isFocusWithin, setIsFocusWithin] = useState<boolean>(false);
+
+  const intl = useIntl();
+
+  const { pressProps } = usePress({
+    onPress: () => onClick(accountTypeId),
+  });
+
+  const { focusWithinProps } = useFocusWithin({
+    onFocusWithinChange: setIsFocusWithin,
+  });
+
+  return (
+    <div
+      className={cx({
+        'flex flex-col items-center gap-3 px-12 py-8 text-center bg-white border rounded-xl': true,
+        'cursor-pointer hover:ring-1 hover:ring-green-dark transition-all': true,
+        'ring-2 ring-green-dark': isFocusWithin,
+      })}
+      role="group"
```

Comment:
Is that role useful? If so, I'd be curious to read more about it.

Reply:
When navigating with VoiceOver, it makes it so that the elements inside are considered part of a group. In this case, the goal was so that someone navigating using a screen reader would be aware that the illustration, button and description inside this element are all part of the same "group" of elements (which in this case a card). 

However, I tested it again with VoiceOver and it turns out it's not needed. By setting the `aria-label` VoiceOver does it automatically (does announce the card's items as a group), so I removed the role.

Reply:
Thanks for the explanation!